### ISSUE TITLE
Enable forwarding between wireless networks

### ISSUE THREAD

**Issue by danielhjames** [64studio/plegger#2]
In  /etc/sysctl.conf file:
`# Uncomment the next line to enable packet forwarding for IPv4`
`net.ipv4.ip_forward=1`

Then update while running:
`echo 1 > /proc/sys/net/ipv4/ip_forward`

Add a route to wlan1 on the upstream Internet side
`iptables -t nat -A POSTROUTING -o wlan1 -j MASQUERADE`

**Comment by obbardc** [comment]
See file webapp/wifi-nat-forwarding-notes.txt for other forwarding for ethernet/wan forwarding

**Comment by danielhjames** [comment]
Thanks @obbardc ! Also I see we can use sysctl -w net.ipv4.ip_forward=1

**Comment by obbardc** [comment]
@danielhjames Right, but that only enables it for the current session.  Editing the sysctl.conf file saves the setting every boot. I think there is a sysctl.conf.d folder, somewhere?
